main.js

```diff
@@ -0,0 +1,68 @@
+import * as Papi from "./ProductService.js";
+import * as Aapi from "./ArticleService.js";
+
+class Product {
+  constructor({
+    name = "상품 이름",
+    description = "string",
+    price = 0,
+    tags = ["전자제품"],
```

Comment:
태그의 기본값은 항상 전자제품인가요?
그렇다면 Product 인스턴스 생성 시  tags가 없다면 항상 ElectronicProduct처럼 보이는 Product 인스턴스가 생성될거에요. 뭔가 논리적으로 맞지 않는 것 같아요. 빈 배열이나 다른 기본값이 적절할 것 같아요.

```suggestion
    tags = [],
```